### PR TITLE
Add addTuteeCommand, Tutor, Tutee tests

### PR DESCRIPTION
**Pull request by kohkakohla**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/AddTuteeCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,242 @@
+package seedu.address.logic.commands;
```
Comment (mingyang143) [inline src/test/java/seedu/address/logic/commands/AddTuteeCommandTest.java:1]: I like that you followed the naming convention for each test case.

File: src/test/java/seedu/address/model/person/TutorTest.java [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+    @Test
+    public void toStringMethod() {
+        String expected = Tutor.class.getCanonicalName()
+                + "{name=" + ALICE.getName() + ", phone=" + ALICE.getPhone()
+                + ", email=" + ALICE.getEmail() + ", address=" + ALICE.getAddress() + ", hours=" + ALICE.getHours()
+                + ", tags=" + ALICE.getTags() + "}";
+        assertEquals(expected, ALICE.toString());
+    }
+}
```
Comment (mingyang143) [inline src/test/java/seedu/address/model/person/TutorTest.java:107]: LGTM

**Review by mingyang143 — CHANGES_REQUESTED** [review]
Do correct the merge conflicts locally first before we can merge.

File: src/test/java/seedu/address/logic/commands/AddTuteeCommandTest.java (hunk 1 above)
Comment (kohkakohla) [reply src/test/java/seedu/address/logic/commands/AddTuteeCommandTest.java:1]: Thanks, Mingyang!

File: src/test/java/seedu/address/logic/commands/AddTuteeCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+                addTuteeCommand.execute(modelStub, commandHistory));
+    }
+
+    @Test
+    public void equals() {
+        Tutee alice = new TuteeBuilder().withName("Alice").build();
+        Tutee bob = new TuteeBuilder().withName("Bob").build();
+        AddTuteeCommand addAliceCommand = new AddTuteeCommand(alice);
+        AddTuteeCommand addBobCommand = new AddTuteeCommand(bob);
+
```
Comment (Plishh) [inline src/test/java/seedu/address/logic/commands/AddTuteeCommandTest.java:70]: Good comments to differentiate different cases

File: src/test/java/seedu/address/model/person/TuteeTest.java [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+public class TuteeTest {
+
+    @Test
+    public void asObservableList_modifyList_throwsUnsupportedOperationException() {
+        Tutee tutee = new TuteeBuilder().build();
+        assertThrows(UnsupportedOperationException.class, () -> tutee.getTags().remove(0));
+    }
+
+    @Test
+    public void isSamePerson() {
```
Comment (Plishh) [inline src/test/java/seedu/address/model/person/TuteeTest.java:29]: good comments to differentiate different cases

**Review by Plishh — APPROVED** [review]
LGTM nice amount of test cases

**Pull request merged**